build/build.sh

```diff
@@ -43,3 +44,20 @@ wkhtmltopdf \
   --quiet \
   output/index.html \
   output/manuscript.pdf
+
+echo "Exporting Word Docx manuscript"
```

Comment:
I'm not sure we want the word export as part of every build. Primarily do we want to track these files on GitHub which will add to repo size? Note you can always use git to go back to a certain point in the repos history and generate one of these docs.

Do we instead want to make it optional via some sort of argument / environment variable? Or even put in a different script?